packages/venia-concept/src/index.js

```diff
@@ -32,11 +32,20 @@ const authLink = setContext((_, { headers }) => {
     const storage = new BrowserPersistence();
     const token = storage.getItem('signin_token');
 
+    if (!storage.getItem('store_view')) {
+        storage.setItem('store_view', {
+            code: STORE_VIEW_CODE,
+            locale: STORE_VIEW_LOCALE
+        });
+    }
+
+    const storeCode = storage.getItem('store_view').code;
```

Comment:
Remove the default state above, preferably.

```suggestion
    const storeCode = storage.getItem('store_view').code || STORE_VIEW_CODE;
```